Sources/PrimerSDK/Classes/Data Models/Apaya.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  Apaya.swift
+//  PrimerSDK
+//
+//  Created by Carl Eriksson on 27/07/2021.
+//
+
+public struct ApayaCreateSessionAPIRequest: Encodable {
```

Comment:
This could be refactored in sth like the following. I think it's more readable.

```
struct Apaya {
    struct CreateSessionAPIRequest {
        // ...
    }

    struct CreateSessionAPIResponse {
        // ...
    }
}
```

and be used as:

```
Apaya.CreateSessionAPIRequest()
```

Reply:
nice!